src/components/button/Button.js

```diff
@@ -1 +1,80 @@
 import styled from 'styled-components';
+
+const Button = (props) => {
+
+    const Button = styled.div`
+        display: table;
+        user-select: none;
+        border-radius: 8px;
+        padding: 0.8rem 1.5rem;
+
+        font-family: "", sans-serif;
+        font-weight: bold;
+
+        transition: all 0.13s linear; 
+        position: relative;
+        overflow: hidden;
+        cursor: pointer;
+        
+        position: relative;
+
+        ${p => p.active 
```

Comment:
I would suggest making separate methods for getting background, active, etc styles as this isn't the easiest to read. So for background styles something like:
`  const getBackgroundStyles = (p) => {
    if (p.variant === "primary") {
      return "background: linear-gradient(to bottom right, #0DEFE1, #78FF96); color: #2C2543; ";
    } else if (p.variant === "secondary") {
      return "background: #FFF;";
    } else {
      return "background: #BDBAC3; color: #8C898F; cursor: default;";
    }
  };`

with  ` ${(p) => getBackgroundStyles(p)}` inside the styled div.

Reply:
that makes a whole lot of sense! Thank you